Assets/Scripts/LifeManager.cs

```diff
@@ -85,6 +120,12 @@ public void Respawn(Vector3 respawnPos)
             mAttackScript.enabled = true;
             mCollider.enabled = true;
             mWeaponInventory.EquipWeapon(GameConstants.WeaponId.FISTS);
+
+            if (mIsLocalPlayer)
```

Comment:
this is an example of a place where we SHOULD be guarding against local player because theres code that all clients need to execute anyway